Mathlib/Topology/Algebra/Group/Basic.lean

```diff
@@ -1252,6 +1252,48 @@ theorem subset_interior_smul : interior s • interior t ⊆ interior (s • t)
 
 end ContinuousConstSMul
 
+section ContinuousSMul
+
+variable [TopologicalSpace α] [TopologicalSpace β] [Group α] [MulAction α β] [ContinuousInv α]
+  [ContinuousSMul α β] {s : Set α} {t : Set β}
+
+/- One may expect a version of the following lemma where `t` is compact and `s` is closed,
+but to do that we would need to ensure that, if `x` is fixed, then we can continuously
+map each `y ∈ s • {x}` to some `g ∈ s` such that `y = g • x`. One case where this is true is
+for `NormedAddTorsor`s, but we don't have a really nice way to state it more generally. -/
```

Comment:
In fact there can't be any reasonable such lemma: for a counterexample, consider Q acting on R by translation, and `s = univ` (which is closed) and `t= {0}` which is compact, but `s • t` is not closed. The issue is that compactness is somewhat intrinsic (the image of a compact set under an embedding is always compact) while closedness is not. Note that this counterexample is also a counterexample to your `NormedAddTorsor` claim, if I understand correctly, so the comment might be worth updating.

Reply:
That's a nice counterexample! I will include it in the comment.
Indeed my current claim is partially false, but I think it is still true for `NormedAddTorsor` (R is not an add torsor on Q right? Because the action is not transitive). I think the correct conditions are:
- there is a continuous map from `s • {x}` to `s` such which, given a `y`, finds a `g` such that `y = g • x`. This is true for `NormedAddTorsor` since `VSub` is continuous.
- each `s • {x}` is closed! This is also true for `NormedAddTorsor` since `s • {x}` is the preimage of `s` under `y ↦ vsub y x`, but this is clearly what fails in your example. 

I'm pretty sure a similar proof would work under these hypotheses but I don't think it's useful. I think I'll add the `NormedAddTorsor` version though, so that we have a definitive answer on this side.